hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -436,7 +436,7 @@ public TableDescriptorBuilder removeColumnFamily(final byte[] name) {
     return this;
   }
 
-  public TableDescriptorBuilder removeCoprocessor(String className) {
+  public TableDescriptorBuilder removeCoprocessor(String className) throws IOException {
```

Comment:
The addition of exceptions to these method signatures breaks backward compatibility of this `IA.Public` class.

Reply:
oh, this is a good point. So, if I use IllegalArgumentException as a runtime exception and remove this signature of IOException, will that be fine?